lib/Crate.js

```diff
@@ -5,7 +5,8 @@ var os = require('os'),
   mmm = require('mmmagic'),
   Magic = mmm.Magic,
   check = require('check-types'),
-  FileProcessor = require('./FileProcessor')
+  FileProcessor = require('./FileProcessor'),
+  targetBuilder = require('./targetBuilder')
```

Comment:
Be careful with your casing - the new file is `TargetBuilder` but you are importing `targetBuilder` which will work on case-insensitive file systems (e.g. Windows, Mac OS X) but fail on case sensitive ones (e.g. most Linux file systems)